scripts/scil_compute_bundle_profiles.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute tract profiles and their statistics along streamlines.
```

Comment:
This script is called bundle_profiles. Please replace all tract profiles for bundle profiles in the docstring.